code/modules/mob/mob_movement.dm

```diff
@@ -135,6 +135,10 @@
 			direct = newdir
 			n = get_step(L, direct)
 
+	if(HAS_TRAIT(L, TRAIT_BEFUDDLED))
+		direct = turn(direct, 180)
+		n = get_step(L, direct)
```

Comment:
Not a fan of adding more checks to alter movement, the confusion check is right over this one, nor of adding a status effect that's just reliably inverting the control scheme.

Reply:
I think the befuddled status effect is conceptually better than confusion due since the power of the effect is dependant on the skill of the afflicted player rather than RNG.

Reply:
The ability to navigate on inverted controls is not one I value highly on our kind of game, personally.
Same for stuff like pixel hunting and several others, to be clear.

Reply:
Yea, the more I think about this, the more I dislike it.
Firstly, it's an extra check in the movement code, every single time something moves, instead of a signal interception. That's bed in itself.
Besides that, a feature that can be instantly negated by inverting your control scheme is not ideal.
Please change this to add confusion or whatever effect other than this one.

Reply:
I have reworked this.